dojox/1.11/gfx.d.ts

```diff
@@ -0,0 +1,289 @@
+/// <reference path="gfx/arc.d.ts" />
+/// <reference path="gfx/bezierutils.d.ts" />
+/// <reference path="gfx/decompose.d.ts" />
+/// <reference path="gfx/filters.d.ts" />
+/// <reference path="gfx/fx.d.ts" />
+/// <reference path="gfx/gradient.d.ts" />
+/// <reference path="gfx/gradutils.d.ts" />
+/// <reference path="gfx/matrix.d.ts" />
+/// <reference path="gfx/Moveable.d.ts" />
+/// <reference path="gfx/Mover.d.ts" />
+/// <reference path="gfx/path.d.ts" />
+/// <reference path="gfx/registry.d.ts" />
+/// <reference path="gfx/shape.d.ts" />
+/// <reference path="gfx/utils.d.ts" />
+/// <reference path="gfx/VectorText.d.ts" />
+
+declare namespace dojox {
+	namespace gfx {
+		type ColorLike = dojo._base.ColorValue | dojo._base.ColorValueAlpha | dojo._base.ColorObject | string;
+		type CubicBezierCurve = [number, number, number, number, number, number, number, number];
+		type Fill = ColorLike | LinearFill | RadialFill | Pattern;
+		type GfxElement = Group | dojox.gfx.shape.Shape | dojox.gfx.shape.Surface;
+		type QuadraticBezierCurve = [number, number, number, number, number, number];
+
+		interface CreateSurface {
+			(parentNode: Node, width: number | string, height: number | string): dojox.gfx.shape.Surface;
+		}
+
+		interface FixTarget {
+			(event: Event, target: GfxElement): boolean;
+		}
+
+		interface Font extends SimpleShape {
+			family?: string;
+			size?: string;
+			style?: string; // TODO: enum
+			variant?: string; // TODO: enum
+			weight?: string; // TODO: enum
+		}
+
+		interface GradientOffsetColor {
+			color: string;
+			offset: number;
+		}
+
+		interface Group extends dojox.gfx.shape.Shape, dojox.gfx.shape.Container, dojox.gfx.shape.Creator {
+			setRawNode(node: Node): void;
+		}
+
+		interface GroupConstructor extends dojox.gfx.shape.ShapeConstructor {
+			new(): Group;
+			prototype: Group;
+		}
+
+		interface LinearFill {
+			colors: string[] | dojo._base.Color[];
+			type: string; // TODO: enum
+			x1: number;
+			x2: number;
+			y1: number;
+			y2: number;
+		}
+
+		interface LinearGradient extends SimpleShape {
+			angle?: number;
+			colors?: GradientOffsetColor[];
+			x1: number;
+			x2: number;
+			y1: number;
+			y2: number;
+		}
+
+		interface Pattern extends SimpleShape {
+			height?: number;
+			src: string;
+			width?: number;
+			x?: number;
+			y?: number;
+		}
+
+		interface Point {
+			x: number;
+			y: number;
+		}
+
+		interface RadialFill {
+			colors?: string[] | dojo._base.Color[];
+			cx?: number;
+			cy?: number;
+			r?: number;
+			type?: string; // TODO: enum
+		}
+
+		interface RadialGradient extends SimpleShape {
+			colors?: GradientOffsetColor[];
+			cx: number;
+			cy: number;
+			r: number;
+		}
+
+		// TODO: maybe complete this? (probably not important)
```

Comment:
Can we update this comment to list what is missing and why we're leaving it off (e.g. renderers are mostly an internal construct). That way if somebody does need this they at least know why it's incomplete.